tools/grab_micro_posts.py

```
@@ -7,16 +7,22 @@
 
 # Load environment variables from .env file if it exists
 load_dotenv()
 
+# Centralized logging configuration
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+
 def download_json_feed(url):
-    response = requests.get(url)
-    if response.status_code == 200:
+    try:
+        response = requests.get(url)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)